provisioning/tasks/build-makefile.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Copy drush makefile into place.
+  copy:
+    src: "../../{{ drush_makefile_path }}"
```

Comment:
Can we just include the `../../` in the path and that way people can specify absolute or relative paths to use?